vault-admin-client/src/test/java/com/scality/osis/vaultadmin/impl/VaultAdminImplTest.java

```diff
@@ -23,6 +29,9 @@
 
 @SuppressWarnings("ConstantConditions")
 public class VaultAdminImplTest extends BaseTest {
+
+
+    public static final String CD_TENANT_ID_PREFIX = "cd_tenant_id%3D%3D";
```

Comment:
It might help with code readability. `cd_tenant_id` is enough :)

Reply:
Sure I will change it

Reply:
Updated in the field and corrected it to `==` as java is decoding the html escape chars